### PR TITLE
SONARIAC-1346 Highlight precise array values in Helm expression in primary location

### PR DESCRIPTION
**Pull request by mstachniuk**

### CODE REVIEW

File: iac-extensions/kubernetes/src/test/resources/checks/CapabilitiesCheck/CapabilitiesChart/templates/capabilities-pod.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+      securityContext:
+        capabilities:
+# Noncompliant@+1
+          add: {{ .Values.capabilities | toYaml | nindent 10 }}
+#                        ^^^^^^^^^^^^^^^^^^^^
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  name: test-caps1
```
Comment (petertrr) [inline iac-extensions/kubernetes/src/test/resources/checks/CapabilitiesCheck/CapabilitiesChart/templates/capabilities-pod.yaml:19]: Is this test duplicating the first one? Should it be `capabilities2`?

File: iac-extensions/kubernetes/src/test/resources/checks/CapabilitiesCheck/CapabilitiesChart/templates/capabilities-pod-secondary.yaml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+      command: ["sleep", "6000"]
+      name: test-container
+      securityContext:
+        capabilities:
+          add: {{ .Values.capabilities | toYaml | nindent 10 }}
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  name: test-caps1
```
Comment (petertrr) [inline iac-extensions/kubernetes/src/test/resources/checks/CapabilitiesCheck/CapabilitiesChart/templates/capabilities-pod-secondary.yaml:17]:
```suggestion
  name: test-caps2
```

**Review by petertrr — APPROVED** [review]
LGTM

File: iac-extensions/kubernetes/src/test/resources/checks/CapabilitiesCheck/CapabilitiesChart/templates/capabilities-pod.yaml (hunk 1 above)
Comment (mstachniuk) [reply iac-extensions/kubernetes/src/test/resources/checks/CapabilitiesCheck/CapabilitiesChart/templates/capabilities-pod.yaml]: ok fixed names.

**Pull request merged**